dashboard/optional_utils.py

```diff
@@ -268,8 +267,6 @@ async def decorator(self, *args, **kwargs):
                         ray.shutdown()
                         raise e from None
 
-                if connect_to_serve:
-                    serve.start(detached=True, _override_controller_namespace="serve")
```

Comment:
why move out of decorator?

Reply:
Resolved offline; I thought the decorator ran only once per function, but it runs once per call. I moved `serve.start()` back into the decorator.